charts/newrelic-infrastructure/README.md

```diff
@@ -1,29 +1,102 @@
+pod near the KSM to reduce the inter-node traffic.
+
+Take a look to the [`values.yaml`](values.yaml) so see how to configure them if you are having problems scheduling pods where you want to.
+
+### `hostNetwork` toggle
+
+In versions below v3, changing the `privileged` mode affected the `hostNetwork`. We changed this behavior and now you can set pods to use `hostNetwork`
+using the corresponding [flags from the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md)
+(`.global.hostNetwork` and `.hostNetwork`) but the component that scrapers data from the control plane has always set `hostNetwork` to true (Look in the
+[`values.yaml`](values.yaml) for `controlPlane.hostNetwork: true`)
+
+This is because the most common configuration of the control plane components is to be configured to listen only to `localhost`.
+
+If your cluster security policy does not allow to use `hostNetwork`, you can disable it control plane monitoring by setting `controlPlane.enabled` to
+`false.`
+
+### `privileged` toggle
+
+The default value for `privileged` [from the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md) is
+`false`.
+
+In this chart it is set to `true` (Look in the [`values.yaml`](values.yaml) for `privileged: true`) because it set `kubelet` pods (the ones that scrape
+metrics from the hosts itself) into privileged mode so it can fetch more fine-grained cpu, memory, process and network metrics for your nodes.
+
+If your cluster security policy does not allow to to have `privileged` in your pod' security context, you can disable it by setting `privileged` to
+`false.`
 
 ## Values
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | common | object | See `values.yaml` | Config that applies to all instances of the solution: kubelet, ksm, control plane and sidecars. |
 | common.agentConfig | object | `{}` | Config for the Infrastructure agent. Will be used by the forwarder sidecars and the agent running integrations. See: https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/ |
-| common.config.interval | duration | `15s` if `lowDataMode == false`, `30s` otherwise. | Intervals larger than 40s are not supported and will cause the NR UI to not behave properly. Any non-nil value will override the `lowDataMode` default. |
+| common.config.interval | duration | `15s` (See [Low data mode](README.md#low-data-mode)) | Intervals larger than 40s are not supported and will cause the NR UI to not behave properly. Any non-nil value will override the `lowDataMode` default. |
+| containerSecurityContext | string | `nil` | This is still to be decided in a follow-up PR regarding privileged mode and defaults changing in an evil way |
```

Comment:
Shouldn't we decide this before publishing? 

Anyway, I would not put that into the readme, but in a issue.

Being a user I would be scared reading that

Reply:
It was a leftover. This is completely defined in the previous PR. This task was also meant to change this but I overlooked it.

Reply:
Solved in [#0fb8df2](https://github.com/newrelic/nri-kubernetes/pull/409/commits/0fb8df26c7f339111945dbe2bd0c2af63943cbc0)